config/kubernetes/manager_webhook_patch.yaml

```diff
@@ -21,3 +21,11 @@ spec:
         secret:
           defaultMode: 420
           secretName: webhook-server-cert
+---
+# The following patch adds a directive for certmanager to inject CA into the CRD
```

Comment:
isn't better/cleaner to have this in its own patch file and let `kustomization` patch it ?

Reply:
Sure, it's just a preference thing I can do that if you'd like, I wish I could just point to `../crd/patches/cainjection_in_flowcollectors.yaml` but it won't let me point to that file from the `kubernetes/` directory

Reply:
just wanted to keep the original sdk yamls as close as the original auto generated ones so any future interactions with sdk-operator will be smooth

Reply:
I can do that later if needed

Reply:
All good I can do it now

Reply:
Done

Reply:
Thanks!
/lgtm